webextensions/api/extension.json

```
@@ -218,7 +218,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/extension/inIncognitoContext",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "8"
```

Previously, https://github.com/chromium/chromium/commit/b2a74caaf15e234b5e6c03151d12c28a319a06cb in [5.0.353.0](https://github.com/chromium/chromium/releases/tag/5.0.353.0) (before 5.0.375) implemented this feature as `inIncognitoTab`, which was only available in manifest v1, and https://github.com/chromium/chromium/commit/21b9dfa740b4c4788c07185e88a6dac0d0a22646 removed in [67.0.3381.0](https://github.com/chromium/chromium/releases/tag/67.0.3381.0) before 67.0.3396.

:x: https://github.com/chromium/chromium/commit/bc535ee5bb4eece29f5d88bcc688613b3b208b27 in [7.0.512.0](https://github.com/chromium/chromium/releases/tag/7.0.512.0) before 7.0.517.
```suggestion
                "version_added": "7"
```

---

@queengooborg Should we document the alternative name `inIncognitoTab` for history?